src/crimson/os/seastore/object_data_handler.cc

```diff
@@ -94,7 +94,6 @@ ObjectDataHandler::write_ret do_insertions(
       LOG_PREFIX(object_data_handler.cc::do_insertions);
       if (region.to_write) {
 	assert_aligned(region.addr);
-	assert_aligned(region.len);
```

Comment:
Here as well, don't all callers ensure that the regions are aligned?

Reply:
sure, will do aligned in caller.

Reply:
```
assert_aligned(region.len);
ceph_assert(region.to_write->length() == region.len);
```